spec/operatoroverloading.dd

```diff
@@ -514,6 +515,23 @@ struct S
         version of $(D opEquals).  Otherwise, inequalities like $(D a <= b)
         will behave inconsistently with equalities like $(D a == b).)
 
+$(SPEC_RUNNABLE_EXAMPLE_RUN
+---
+struct S
+{
+    int i, j;
+    int opCmp(ref const S s) const { return i - s.i; } // ignore j
```

Comment:
I don't think it's a good example, since it doesn't account for overflow. I suggest: `return (i > s.i) - (i < s.i);`

Reply:
Now fixed